README.md

```diff
@@ -58,6 +58,4 @@ This project part of a Service-Oriented Architecture (SOA) application. The Weat
     
 #### Will Dunlap: [LinkedIn](https://www.linkedin.com/in/willwdunlap/), [Email](dunlapww@gmail.com), [GitHub](dunlapww)
```

Comment:
This README looks awesome! Thanks for adding links for each of us!

Reply:
Thank you for putting this together George! It looks like the Github links are broken here, but should be an easy fix.

Reply:
Thanks for the template George, I used this template for the FE Readme as well.